doc/combining.rst

```diff
@@ -237,3 +238,75 @@ coordinates as long as any non-missing values agree or are disjoint:
 Note that due to the underlying representation of missing values as floating
 point numbers (``NaN``), variable data type is not always preserved when merging
 in this manner.
+
+.. _combining.multi:
+
+Combining along multiple dimensions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+
+  There are currently three combining functions with similar names:
+  ``auto_combine``, ``combine_auto``, and ``combine_manual``. This is because
+  ``auto_combine`` is in the process of being deprecated in favour of the other
+  two functions, which are more general. If your code currently relies on
+  ``auto_combine``, then you will be able to get similar functionality by using
+  ``combine_manual``.
+
+For combining many objects along multiple dimensions xarray provides
+``combine_manual`` and ``combine_auto``. These functions use a combination of
```

Comment:
To turn these references into links to the API documentation, you can write something like:

    :py:func:`~xarray.combine_manual`

which will print like [`combine_manual`](http://link-to-doc).